app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,3 @@
 // Custom bootstrap variables must be set or imported *before* bootstrap.
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
```

Comment:
we're not using this file as we're not using sprockets.  Please move it to app/javascripts (for webpacker)

Reply:
Yeah, working on that. Missed it before I moved out of draft. Thanks.